pkgs/development/tools/gojq/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub, testVersion }:
+let self = buildGoModule rec {
+  pname = "gojq";
+  version = "0.12.6";
+
+  src = fetchFromGitHub {
+    owner = "itchyny";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-hdrxd1eyr5y2G1LsQgmih7H6I3Dsbo0qmXr2WhpeghQ=";
+  };
+
+  vendorSha256 = "sha256-1wN1nMpvkkrKISuI9tCxX8nL73TQr9snHS9A1UWCArQ=";
+
+  CGO_ENABLED = 0;
+
+  ldflags = [ "-s" "-w" ];
+
+  passthru.tests.version = testVersion {
+    package = self;
```

```suggestion
    package = gojq;
```
You can instead add gojq as an argument to the derivation, even if it looks a bit weird - equally as the let in hackery...